frontend/src/common/components/event-form/event.form.jsx

```diff
@@ -44,16 +49,25 @@ const EventFrom = ({ onSubmit, initialValues }) => {
 
   const validationSchema = yup.object().shape({
     title: yup.string().required(texts.title_required),
-    phone: yup.string().required(texts.phone_required),
+    phone: yup
+      .string()
+      .matches(regexStrings.phone, texts.phone_invalid)
+      .required(texts.phone_required),
     room: yup.array().min(1, texts.room_required),
     description: yup.string(),
     start: yup.date().required(),
     end: yup.date().required(),
     isActivity: yup.bool().required(),
     permit: yup.bool(),
     responsible_name: yup.string().when("isActivity", whenTrue),
-    responsible_number: yup.string().when("isActivity", whenTrue),
-    responsible_email: yup.string().when("isActivity", whenTrue),
+    responsible_number: yup
+      .string()
+      .matches(regexStrings.phone, texts.phone_invalid)
+      .when("isActivity", whenTrue),
+    responsible_email: yup
```

Comment:
If I have not submitted a party report, it complains that the email is invalid of the responsible person.
The yup validation check should only be run for the party report credentials if a party report has been submitted